Procurement/Controls/SetBuyoutView.xaml

```diff
@@ -52,8 +51,14 @@
 
 
         <TextBlock Grid.Row="3" Foreground="#FFAB9066" Text="Notes:" />
+
+        <TextBlock Grid.Row="3" Foreground="#FFAB9066" Text="POB Data copied to Clipboard" Grid.Column="1" Grid.ColumnSpan="2" Visibility="{Binding IsDataInClipboard, Converter={StaticResource bc}}"/>
+
+        <Button Content="Export to POB" ToolTip="Copy item data to clipboard in Path of Building format" 
+                Grid.Column="2" Height="25" Command="{Binding PobDataGenerationCommand}"  Grid.Row="3" Visibility="{Binding IsDataInClipboard, Converter={StaticResource bc}, ConverterParameter=Invert}"
```

Comment:
I don't like having the button disappear after the user presses it.  It doesn't appear clear to the user why it disappears, and when it will be back.  It doesn't hurt anyone for the user to just continue pressing the button, so we shouldn't limit him from doing so.

Reply:
The reason I hid the button is that there is limited space in the view and I didn't want to make too many changes here.